public/datasets/movies.json

```diff
@@ -60,7 +72,16 @@
       "id": "Star",
       "description": "The lead actor or actress."
     },
-    { "type": "categorical", "id": "Writer", "description": "" },
+    {
+      "type": "numerical",
+      "id": "Votes",
+      "description": ""
+    },
+    {
+      "type": "categorical",
+      "id": "Writer",
+      "description": ""
+    },
```

Comment:
Is it worth adding descriptions for columns while you're in here? Or is that already on deck for @moneppo ?

Reply:
As discussed with @moneppo, he'll add descriptions in his next PR :)